client/signup/steps/jpo-site-type/select-business-personal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { translate } from 'i18n-calypso';
+import Card from 'components/card';
+import Button from 'components/button';
+
+import PersonalGraphic from './personal-graphic';
+import BusinessGraphic from './business-graphic';
+
+module.exports = React.createClass( {
+
+	displayName: 'SelectBusinessPersonal',
+
+	propTypes: {
+		required: React.PropTypes.bool,
+	},
+
+	render: function() {
+		if ( ! this.props.current ) {
+			return ( <div /> );
+		}
+
+		return ( 
+			<div className="jpo__site-type-wrapper">
+				<div className="jpo__site-type-row jpo__site-type-row-small">
+					<Card>
+						<PersonalGraphic />
+						<Button onClick={ this.props.onSelectPersonal }>{ translate( 'Personal site' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Personal Site' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).